client/wfm/src/executor/Executor.ts

```diff
@@ -1,5 +1,14 @@
+import * as Promise from 'bluebird';
 import {ProcessInstance} from '../process-instance/ProcessInstance';
 import {Process} from '../process/Process';
+import {Task} from '../task/Task';
+
+export interface TaskMoveResponse {
```

Comment:
Can we work with documentation first aproach and add ts-docs compatiblie documentation when creating classes.  I'm not sure what is the purpose of this class for the moment.